doc/user_guide/bank/essentials_calendar.md

```diff
@@ -1 +1,34 @@
-Not yet written (We have someone working on it though as of 20240719)
\ No newline at end of file
+# Pick Ups & Deliveries Calendar
+
+The calendar shows the pickup and delivery schedule. It shows past and upcoming distributions in a given month.
```

Comment:
Pick Ups & Deliveries shows scheduled and completed distributions in a calendar format on a monthly basis.

[Suggested change tightens things up a bit - and removes having the verb "show" twice in the same line.]